CHANGELOG-3.1.md

```diff
@@ -0,0 +1,25 @@
+# CHANGELOG for 3.1.x
+
+## 3.1.0 (2022-06-19)
+
+### PHP Versions
+
+  * 🗑️ Drop PHP 7.2 support
```

Comment:
Codacy has a fix for the issue: [Expected: 0; Actual: 2](https://app.codacy.com/gh/yoanm/symfony-jsonrpc-http-server/pullRequest?prid=9640734)
```suggestion
* 🗑️ Drop PHP 7.2 support
```